core/dataflow/src/main/resources/udf_server_main.py

```diff
@@ -0,0 +1,185 @@
+import sys
+from traceback import print_exc
+
+import pandas
+import pyarrow
+import ast
+import threading
+import pyarrow.flight
+import importlib.util
+
+from pyarrow.lib import ArrowTypeError
+
+import udf_operator
+
+portNumber = sys.argv[1]
+UDFOperatorScript = sys.argv[2]
+
+# Dynamically import operator from user-defined script.
+spec = importlib.util.spec_from_file_location('user_module', UDFOperatorScript)
+user_module = importlib.util.module_from_spec(spec)
+spec.loader.exec_module(user_module)
+FinalUDF = None
```

Comment:
add a comment explain what of these global variables do

Reply:
I'll add the comment.

Reply:
Done.